commands/addons/create.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function parseConfig (args) {
```

Comment:
Would this be best handled in a module somewhere? Parsing CLI args is surprisingly nuanced, depending on how advanced you get.